src/WebAppSSLManager/Models/Settings.cs

```diff
@@ -75,6 +76,12 @@ public static void Init(ILogger logger)
                 _logger.LogWarning("EmailSender environment variable is null. Reverting to default");
                 EmailSender = Constants.DefaultEmailSender;
             }
+
+            var hasValue = bool.TryParse(Environment.GetEnvironmentVariable("UseStaging"), out var useStaging);
```

Comment:
can you make this in-line in the if instead of using the "has value" variable?

Also please remove the { } from the if :)

Reply:
I removed the if statement completely as the result will be false anyway even if the parse fails.